### PR TITLE
Bubble sort project in Ruby for Microverse

### PR DESCRIPTION
**Pull request by tiagoit**
Bubble sort project in Ruby for Microverse

### CODE REVIEW

File: src/bubble_sort.rb [hunk 1]
```diff
@@ -0,0 +1,29 @@
+#!/usr/bin/env ruby
+# frozen_string_literal: true
+
+def bubble_sort(array)
+  array_ = array # method should not be destructive
+
+  array_.length.times do |j|
+    (array_.length - j).times do |i| # loop till the not ordered element, on first run, last element is already ordered
+      yield array, i
+    end
```
Comment (Oghenebrume50) [inline src/bubble_sort.rb:12]: bubble_Sort function should just sort in descending order, it should not be passed a block.

**Review by Oghenebrume50 — CHANGES_REQUESTED** [review]
Hi Team, good job. :+1:  I left a comment below please check that out. Since you built the other function around bubble_sort, that couldn't be tested. Please make changes and submit for a re-review :pray:

File: src/bubble_sort.rb [hunk 2]
```diff
@@ -0,0 +1,17 @@
...
+    (array_.size - i).times do |j|
+      if array_[j + 1] &&
+         array_[j] > array_[j + 1]
+        array_[j], array_[j + 1] = array_[j + 1], array_[j]
+      end
+    end
+  end
+  array_
+end
+
```
Comment (samgaco) [inline src/bubble_sort.rb:16]: > Now create a similar method called #bubble_sort_by which sorts an array by accepting a block

There should be another function called **bubble_sort_by** in your project

**Review by samgaco — CHANGES_REQUESTED** [review]
Hi  @ferreirati and @fedgut :high_brightness: 

Great readme file :+1: 

- [ ] You have not implemented bubble_sort_by in your project, please address that and submit back for another code review. 


----------------
* ###### STATUS OF THE PROJECT: REQUESTED CHANGES :stop_sign:

**Review by bolah2009 — APPROVED** [review]
## Nice job implementing all suggested changes.

### It's a :+1: from me,

You can merge to master.

**Pull request merged**